src/librados/IoCtxImpl.cc

```diff
@@ -752,7 +752,7 @@ int librados::IoCtxImpl::aio_operate_read(const object_t &oid,
 int librados::IoCtxImpl::aio_operate(const object_t& oid,
 				     ::ObjectOperation *o, AioCompletionImpl *c,
 				     const SnapContext& snap_context, int flags,
-                                     const blkin_trace_info *trace_info)
+                                     const blkin_trace_info *trace_info, const jspan_context *otel_trace)
```

Comment:
you mentioned in commit this will be directed to replace blkin trace to otel, did you find backward compatibility still a concern?

Reply:
> you mentioned in commit this will be directed to replace blkin trace to otel, did you find backward compatibility still a concern?

No. we can safely remove the blkin stuff, but we want to do it right in another PR since it exist in many places, as well the external library and other dependencies.